hwdb.d/60-evdev.hwdb

```diff
@@ -478,6 +478,13 @@ evdev:name:AlpsPS/2 ALPS GlidePoint:dmi:*svnLENOVO:*pvrLenovoideapadY700-14ISK*
  EVDEV_ABS_01=::29
  EVDEV_ABS_35=::27
  EVDEV_ABS_36=::29
+ 
+# Lenovo Ideapad 310S-14ISK
```

Comment:
I suspect the indentation of this and the following line is off:

``
WARNING: evdev is not available
Cannot parse /build/hwdb.d/60-evdev.hwdb: Expected stringEnd, found '#'  (at char 13359), (line:475, col:1)
/build/hwdb.d/60-evdev.hwdb: 0 match groups, 0 matches, 0 properties
```

Reply:
Looks like the intention is OK. But I'm not sure if the coding of these blanks are correct.

It says `Expected stringEnd, found '#'`. It seems that there're some problems with the line ending. Maybe it's because I made this patch on github editor.

I'll check all of this on my own computer after going back.

Reply:
It's what @mrc0mmand said:
> There's a trailing whitespace.